source/tracyd/api.d

```diff
@@ -151,6 +151,36 @@ void ___tracy_emit_gpu_context_name (const ___tracy_gpu_context_name_data);
 
 int ___tracy_connected ();
 
+public import std.format;
+
+mixin template TracyCZoneBase(string ctx, string name, uint color, bool active)
```

Comment:
I didn't like `ctx` is a string parameter since `TracyCZoneEnd` is not requiring a string rather a variable name.